.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Meshery-linkerd",
-	"image": "aisuko/golang:v1.14",
+	"image": "aisuko/red-thirteen:rt0.1.1",
```

Comment:
Can we have layer5 image here?

Reply:
Nice catch, we can remove this fist, I do not have permission to log into the `meshery` docker hub